wenet/dataset/dataset.py

```diff
@@ -260,11 +432,23 @@ def __init__(self,
                  batch_type='static',
                  batch_size=1,
                  max_frames_in_batch=0,
-                 sort=True):
-        ''' Args:
+                 sort=True,
+                 raw_wav=True):
+        """Dataset for loading audio data.
+
+        Attributes::
             data_file: input data file
-            max_length: drop utterance which is greater than max_length
-            min_length: drop utterance which is less than min_length
+                Plain text data file, each line contains following 7 fields,
+                which is split by '\t':
+                    utt:utt1
+                    feat:tmp/data/file1.wav or feat:tmp/data/fbank.ark:30
+                    feat_shape:5.042(in ms) or feat_shape:495,30
```

Comment:
feat_shape:5.042(in ms) or feat_shape:495,30
the demo here is not consistent, change it to 
feat_shape: 4.95(in seconds) or feat_shape:495,80(495 is in frames)

Reply:
changed